diff --git a/node_modules/@react-native-firebase/auth/android/src/main/java/io/invertase/firebase/auth/ReactNativeFirebaseAuthModule.java b/node_modules/@react-native-firebase/auth/android/src/main/java/io/invertase/firebase/auth/ReactNativeFirebaseAuthModule.java
index 06c73c7..7cab5fa 100644
--- a/node_modules/@react-native-firebase/auth/android/src/main/java/io/invertase/firebase/auth/ReactNativeFirebaseAuthModule.java
+++ b/node_modules/@react-native-firebase/auth/android/src/main/java/io/invertase/firebase/auth/ReactNativeFirebaseAuthModule.java
@@ -1080,10 +1080,10 @@ class ReactNativeFirebaseAuthModule extends ReactNativeFirebaseModule {
       if (forceResend && mForceResendingToken != null) {
         PhoneAuthProvider.getInstance(firebaseAuth)
             .verifyPhoneNumber(
-                phoneNumber, 60, TimeUnit.SECONDS, activity, callbacks, mForceResendingToken);
+                phoneNumber, 0, TimeUnit.SECONDS, activity, callbacks, mForceResendingToken);
       } else {
         PhoneAuthProvider.getInstance(firebaseAuth)
-            .verifyPhoneNumber(phoneNumber, 60, TimeUnit.SECONDS, activity, callbacks);
+            .verifyPhoneNumber(phoneNumber, 0, TimeUnit.SECONDS, activity, callbacks);
       }
     }
   }
@@ -1153,7 +1153,7 @@ class ReactNativeFirebaseAuthModule extends ReactNativeFirebaseModule {
         PhoneAuthOptions.newBuilder(firebaseAuth)
             .setActivity(activity)
             .setMultiFactorHint((PhoneMultiFactorInfo) selectedHint)
-            .setTimeout(30L, TimeUnit.SECONDS)
+            .setTimeout(0L, TimeUnit.SECONDS)
             .setMultiFactorSession(resolver.getSession())
             .setCallbacks(
                 new PhoneAuthProvider.OnVerificationStateChangedCallbacks() {
@@ -1198,7 +1198,7 @@ class ReactNativeFirebaseAuthModule extends ReactNativeFirebaseModule {
         PhoneAuthOptions.newBuilder(firebaseAuth)
             .setPhoneNumber(phoneNumber)
             .setActivity(getCurrentActivity())
-            .setTimeout(30L, TimeUnit.SECONDS)
+            .setTimeout(0L, TimeUnit.SECONDS)
             .setMultiFactorSession(multiFactorSession)
             .requireSmsValidation(true)
             .setCallbacks(
