diff --git a/node_modules/react-native-dash/dist/index.js b/node_modules/react-native-dash/dist/index.js
index 4034e31..b780e44 100644
--- a/node_modules/react-native-dash/dist/index.js
+++ b/node_modules/react-native-dash/dist/index.js
@@ -54,12 +54,12 @@ var styles = _reactNative.StyleSheet.create({
 });
 
 Dash.propTypes = {
-	style: _reactNative.ViewPropTypes.style,
+	style: _reactNative.View.prototype || {},
 	dashGap: _propTypes2.default.number.isRequired,
 	dashLength: _propTypes2.default.number.isRequired,
 	dashThickness: _propTypes2.default.number.isRequired,
 	dashColor: _propTypes2.default.string,
-	dashStyle: _reactNative.ViewPropTypes.style
+	dashStyle: _reactNative.View.prototype || {}
 };
 
 Dash.defaultProps = {
